### PR TITLE
[INLONG-10779][Dashboard] Data synchronization adds offline synchronization configuration

### LINKED ISSUE #10779: [Improve][Dashboard] Data synchronization adds offline synchronization configuration
### Description

Data synchronization adds offline synchronization configuration

### InLong Component

InLong Dashboard

### Are you willing to submit PR?

- [X] Yes, I am willing to submit a PR!

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://www.apache.org/foundation/policies/conduct)

### PR DESCRIPTION
**Pull request by wohainilaodou**
Fixes #10779 

### Motivation

Data synchronization adds offline synchronization configuration

### Modifications
A new synchronization type drop-down box is added to the data synchronization page. When adding new data synchronization, you can select the synchronization type, real-time or offline. If you select offline, scheduling rules and configuration-dependent configuration information will appear. The scheduling type can be selected as regular or cron expression, and then Configure related cycles and times. When selecting parallel configuration, you need to enter the maximum number of parallels.

### Verifying this change
before:
image
image

after:
image
image
image
image

### CODE REVIEW

**Review by aloyszhang — APPROVED** [review]

**Review by aloyszhang — DISMISSED** [review]

File: inlong-dashboard/src/plugins/sync/common/SyncDefaultInfo.ts [hunk 1]
```diff
@@ -87,7 +94,170 @@ export class SyncDefaultInfo implements DataWithBackend, RenderRow, RenderList {
...
+  })
+  @I18n('meta.Synchronize.ValidTime')
+  time: dayjs.Dayjs[];
+  @FieldDecorator({
+    visible: values => values.inlongGroupMode === 2 && values.scheduleType === 1,
+    type: 'input',
+    rules: [{ required: true }],
+    props: {},
+  })
+  @I18n('crontab表达式')
```
Comment (leezng) [inline inlong-dashboard/src/plugins/sync/common/SyncDefaultInfo.ts:227]: It seems like locale configuration is missing?

Comment (wohainilaodou) [reply inlong-dashboard/src/plugins/sync/common/SyncDefaultInfo.ts]: done

File: inlong-dashboard/src/ui/locales/cn.json [hunk 2]
```diff
@@ -430,6 +430,7 @@
   "meta.Synchronize.ScheduleUnit.Single": "单次",
   "meta.Synchronize.ScheduleInterval": "调度周期",
   "meta.Synchronize.ValidTime": "有效时间",
+  "meta.Synchronize.CronExpression": "crontab表达式",
```
Comment (dockerzhang) [inline inlong-dashboard/src/ui/locales/cn.json:433]: "crontab表达式"
——>
"Crontab 表达式"

File: inlong-dashboard/src/ui/locales/cn.json [hunk 3]
```diff
@@ -411,7 +411,33 @@
...
+  "meta.Synchronize.ScheduleUnit": "调度单位",
+  "meta.Synchronize.ScheduleUnit.Year": "年",
+  "meta.Synchronize.ScheduleUnit.Month": "月",
+  "meta.Synchronize.ScheduleUnit.Day": "天",
+  "meta.Synchronize.ScheduleUnit.Minute": "分钟",
+  "meta.Synchronize.ScheduleUnit.Hours": "小时",
+  "meta.Synchronize.ScheduleUnit.Single": "单次",
+  "meta.Synchronize.ScheduleInterval": "调度周期",
+  "meta.Synchronize.ValidTime": "有效时间",
+  "meta.Synchronize.CronExpression": "crontab表达式",
```
Comment (dockerzhang) [inline inlong-dashboard/src/ui/locales/cn.json:433]:
```suggestion
  "meta.Synchronize.CronExpression": "Crontab 表达式",
```

Comment (wohainilaodou) [reply inlong-dashboard/src/ui/locales/cn.json]: done

**Review by leezng — APPROVED** [review]

**Review by dockerzhang — APPROVED** [review]

**Pull request merged**